Buildings/Controls/OBC/CDL/Conversions/BooleanToInteger.mo

```diff
@@ -22,8 +22,8 @@ equation
     Documentation(
       info="<html>
 <p>
-Block that outputs the <code>Boolean</code>
-equivalent of the <code>Integer</code> input.
+Block that outputs the <code>Integer</code>
+equivalent of the <code>Boolean</code> input.
 </p>
 <pre>
   y = if u then integerTrue else integerFalse;
```

Comment:
Please also add this in the revision notes of this model, and in the revision notes of `Buildings/package.mo`.

Reply:
Revision and release notes added in 8065511.